changelog/pattern-deprecate.dd

```diff
@@ -8,3 +8,57 @@ $(MREF std, regex) and $(MREF std, algorithm). They will be removed from
 $(MREF std, string) on May 2018.
 
 If you still need to use these, please see $(LINK2 https://github.com/dlang/undeaD, undeaD).
+
+Here's a simple guide on how to migrate your existing code.
+
+$(REF munch, std, string):
+-------
+import std.algorithm;
+import std.string;
+import std.utf;
+
+string s = "\tabc";
+
+// old
+s.munch(whitespace);
+
+// new
+size_t i = s.byCodeUnit.countUntil!(a => !isWhite(a));
```

Comment:
Agreed, but now you need to import `std.functional` (overall more key strokes) :cry: